compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -559,9 +559,15 @@ static Value buildHALWorkgroupInfoOp(OpBuilder &b, unsigned dim) {
 }
 
 linalg::LinalgLoopDistributionOptions getIREELinalgLoopDistributionOptions(
-    const SmallVector<int64_t> &tileSizes) {
-  return {[&tileSizes](OpBuilder &builder, Location loc,
-                       ArrayRef<Range> parallelLoopRanges) {
+    const SmallVector<int64_t> &tileSizes, bool skipDistributionLoops) {
```

Comment:
Instead of the `bool` can we just pass the `linalg::DistributionMethod` value directly?